lib/msal-browser/docs/accounts.md

```diff
@@ -6,10 +6,22 @@
 
 The `@azure/msal-browser` library provides the following APIs to access cached accounts:
 
-* `getAllAccounts()`: returns all the accounts currently in the cache. An application must choose an account to acquire tokens silently.
-* `getAccountByHomeId()`: receives a `homeAccountId` string and returns the matching account from the cache.
-* `getAccountByLocalId()`: receives a `localAccountId` string and returns the matching account from the cache.
-* `getAccountByUsername()`: receives a `username` string and returns the matching account from the cache.
+-   `getAllAccounts()`: returns all the accounts currently in the cache. Supports an optional filter to return a specific set of accounts. An application must choose an account to acquire tokens silently.
+-   `getAccount()`: returns the first cached account that matches the filter passed in. The order in which accounts are read from the cache is arbitrary and there is no guarantee that the first account in the filtered list will be the same for any two calls of `getAccount`. As explained below, increasing the number of filter attributes will provide more exact matches.
+
+### Account Filter Object
+
+The [AccountFilter](https://azuread.github.io/microsoft-authentication-library-for-js/ref/types/_azure_msal_common.AccountFilter.html) type documentation lists the properties that can be used and combined to filter accounts.
```

Comment:
This link's destination doesn't exist yet because the type wasn't public before. If the link doesn't work I'll replace it once type docs are published.